src/main/java/seedu/cafectrl/data/Pantry.java

```diff
@@ -205,16 +191,13 @@ private void handleRestock(Ingredient dishIngredient) {
         ui.showNeededRestock(dishIngredientName, currentQuantity, unit, neededIngredient);
     }
 
-
     /**
      * Retrieves the ingredients for a specific ordered dish.
      *
      * @param orderedDish The name of the ordered dish.
      * @return The list of ingredients for the ordered dish.
      */
     public ArrayList<Ingredient> retrieveIngredientsForDish(String orderedDish){

```

Comment:
Perhaps help to add a whitespace here also
```suggestion
    public ArrayList<Ingredient> retrieveIngredientsForDish(String orderedDish) {

```